test/files/sps004/fail-sps004-scenario01-with_position_with_contain.ifc

```diff
@@ -0,0 +1,38 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2023-09-21T11:25:23',(''),(''),'IfcOpenShell-v0.7.0-64d98f129','IfcOpenShell-v0.7.0-64d98f129','');
+FILE_SCHEMA(('IFC4X3_ADD1'));
```

Comment:
CoordinationView with IFC4X3_ADD1? Intentional?

Reply:
This probably applies to the majority of files. Let's save this for later #91